### ISSUE TITLE
ir_image_tag doesn't accept html options

### ISSUE THREAD

**Issue by jgillman** [jimmynicol/ir-helper#1]
Following the convention of `image_tag` it would be very convenient to be able to pass in html options like `class` or `alt`.

---
#### Current:

``` erb
<%= image_tag( ir_url(my_image_url), class: 'my-great-class', alt: 'My image alt text.' ) %>
or even
<img src="<%= ir_url(my_image_url) %>" class='my-great-class', alt='My image alt text.' />
```
#### Proposed:

``` erb
<%= ir_image_tag( my_image_url, class: 'my-great-class', alt: 'My image alt text.' ) %>
```

---

I realize this is probably easier said than done, but I'm curious to hear your thoughts on it.

**Comment by jimmynicol** [comment]
Take a look at the latest commit to master, it should fix this issue so it mirrors your proposed solution. Basically I was not passing the arguments through to image_tag. Currently it supports `id`, `class`, `style` and `alt.

**Comment by jgillman** [comment]
:+1: Nice! Looks good to me.

Fixed with a81851367311814b5926a3db7bbb15459b185c72
